tests/PySys/log_request_end_to_end/create_test_logs.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/python3
```

Comment:
Wouldn't this force the script to be executed from the system level python instead of the `venv` python used for pysys tests? Not sure if that's what we wanted.

Reply:
Removed this.